dev_mode/index.js

```diff
@@ -64,6 +64,7 @@ export async function main() {
   var deferred = [];
   var ignorePlugins = [];
   var register = [];
+  var entrypoints = []
```

Comment:
I would go for a mapping to store entrypoints by type:

```suggestion
  var entrypoints = {};
```